Sources/ComposableArchitecture/Reducer/Reducers/Observe.swift

```diff
@@ -6,10 +6,17 @@ public struct _Observe<Reducers: ReducerProtocol>: ReducerProtocol {
   ///
   /// - Parameter build: A reducer builder that has access to the current state and action.
   @inlinable
-  public init<State, Action>(
-    @ReducerBuilder<State, Action> _ build: @escaping (State, Action) -> Reducers
+  public init(
+    @ReducerBuilderOf<Reducers> _ build: @escaping (Reducers.State, Reducers.Action) -> Reducers
```

Comment:
We should maybe have a test to make sure this builds. I feel like I had to shadow generics like this for `CombineReducers` to work in some cases, so not sure if the same applies here or if it was just a mistake.